go.mod

```diff
@@ -2,10 +2,12 @@ module github.com/openvex/vexctl
 
 go 1.20
 
+replace github.com/openvex/go-vex => ../go-vex/
```

Comment:
this is correct?

Reply:
Yes for the moment, it will go away once the required chain of previous PRs merges :)  (#110 and #92)

Reply:
Rebased, the local replace is gone